src/main/java/com/rakovets/course/java/core/practice/jcf_set/web_service/User.java

```diff
@@ -0,0 +1,27 @@
+package com.rakovets.course.java.core.practice.jcf_set.web_service;
+
+public class User {
+        private String login;
+        private String password;
+
+    public User(String login, String password) {
+        this.login = login;
+        this.password = password;
+    }
+
+        public String getLogin() {
+        return login;
+    }
+
+        public void setLogin(String login) {
+        this.login = login;
+    }
+        public String getPassword() {
+        return password;
+    }
+
+        public void setPassword(String password) {
+        this.password = password;
+    }
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"